test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/RandomGeneratorTest.java

```diff
@@ -111,35 +129,58 @@ public void shouldGenerateRandomNiNumber() {
         // given
         final Generator<String> niNumberGenerator = RandomGenerator.NI_NUMBER;
 
-        // when
+        // when & then
         typeCheck(niNumberGenerator, s -> s.matches(NI_NUMBER_PATTERN)).verify(times(NUMBER_OF_TIMES));
     }
 
     @Test
-    public void shouldGenerateRandomPostCode() {
+    public void shouldGenerateRandomPostcode() {
         // given
```

Comment:
not sure that the given/when/then needs to be commented. Surely everyone here should understand and recognize it